app/javascript/dashboard/routes/dashboard/settings/reports/components/Heatmap.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <div class="heatmap-wrapper">
+    <div class="heatmap-labels">
+      <div
+        v-for="dateKey in processedData.keys()"
+        :key="dateKey"
+        class="heatmap-axis-label"
+      >
+        {{ getDayOfTheWeek(new Date(dateKey)) }}
+        <time>{{ formatDate(dateKey) }}</time>
+      </div>
+    </div>
+    <div class="heatmap-container">
+      <div
+        v-for="dateKey in processedData.keys()"
+        :key="dateKey"
+        class="heatmap-container-row"
+      >
+        <div
+          v-for="data in processedData.get(dateKey)"
+          :key="data.timestamp"
+          v-tooltip.top="`${data.value} conversations`"
```

Comment:
@pranavrajs @muhsin-k @nithindavid how do I translate text like this?

Reply:
@scmmishra i18n has support for variables. See `app/javascript/dashboard/i18n/locale/en/conversation.json:86` and usage will be like 
```
this.$t(
              'CONVERSATION.CARD_CONTEXT_MENU.API.AGENT_ASSIGNMENT.SUCCESFUL',
              {
                agentName: agent.name,
                conversationId,
              }
            )
          );
```